app/models/application_key.rb

```diff
@@ -2,7 +2,7 @@ class ApplicationKey < ApplicationRecord
 
   KEYS_LIMIT = 5
 
-  APP_KEY_FORMAT = /[a-z\d][a-z\d\-\_]*[a-z\d]/i.freeze
+  APP_KEY_FORMAT = /[\x20-\x7E]+/
```

Comment:
Not freezing it?

Reply:
Same as https://github.com/3scale/porta/pull/963#discussion_r300677463